cloud/installation/poller-remove.md

```diff
@@ -7,7 +7,7 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import MoveResources from './_move-resources.mdx';
 
-> In order to ensure the stability and integrity of your monitoring environment, do not delete the central poller. The central poller plays a critical role in the overall operation of the Centreon platform. Removing it can lead to unintended disruptions in service. We are in the process of implementing safeguards to prevent the accidental deletion of the central poller.
+> In order to ensure the stability and integrity of your monitoring environment, do not delete the "Central" poller. This poller plays a critical role in the overall operation of the Centreon platform. Removing it can lead to unintended disruptions in service. We are in the process of implementing safeguards to prevent the accidental deletion of the "Central" poller.
```

Comment:
```suggestion
> In order to ensure the stability and integrity of your monitoring environment, do not delete the **Central** poller. This poller plays a critical role in the overall operation of the Centreon platform. Removing it can lead to unintended disruptions in service. We are in the process of implementing safeguards to prevent the accidental deletion of the **Central** poller.
```